website/docs/welcome/releases.md

```diff
@@ -8,6 +8,16 @@ title: Releases
 
 Find releases at [https://github.com/microsoft/garnet/releases](https://github.com/microsoft/garnet/releases).
 
+## .NET Tool
+
+Garnet can be installed as a [.NET tool](https://aka.ms/global-tools):
+
+``` txt
+> dotnet tool install --global garnet-server
+>
+> garnet-server
```

Comment:
Just a minor request. Can [website/docs/getting-started/configuration.md](https://github.com/microsoft/garnet/blob/main/website/docs/getting-started/configuration.md) document updated to indicate `garnet-server` (global tools) can also be confirmed using either configuration file or command line arguments.

Currently, it only specifies `GarnetServer.exe`

Reply:
we can take care of these in a separate pass as well. ideally we want to rename GarnetServer.exe to garnet-server.exe so that we only have one concept overall and can just use the `garnet-server` terminology everywhere.